zppy/templates/tc_analysis.bash

```diff
@@ -29,19 +29,13 @@ start="{{ '%04d' % (year1) }}"
 end="{{ '%04d' % (year2) }}"
 caseid="{{ case }}"
 drc_in={{ input }}/{{ input_subdir }}
-# Warning: tempest-remap can only write grid file on SCRATCH space.
-# The result files will be moved to another path at the end.
-result_dir_fin={{ output }}/post/atm # Directory will be {{ output }}/post/atm/tc-analysis_${Y1}_${Y2}
-mkdir -p $result_dir_fin
 y1={{ year1 }}
 y2={{ year2 }}
 Y1="{{ '%04d' % (year1) }}"
 Y2="{{ '%04d' % (year2) }}"
-result_dir={{ scratch }}/tc-analysis_${Y1}_${Y2}/
+result_dir={{ output }}/post/atm/tc-analysis_${Y1}_${Y2}/
```

Comment:
@forsyth2 the changes look good to me. With the fix in mesh generation, I think we no longer need extra step to save file in `scratch`? but it looks like all tc_analysis tests include `scratch`.

Reply:
Yes, that's correct. I just remove the `scratch` parameters from the test `cfg`s in the latest commit.